Cython/Build/Tests/TestCyCache.py

```diff
@@ -1,15 +1,20 @@
-import difflib
 import glob
 import gzip
 import os
 import tempfile
 
-import Cython.Build.Dependencies
-import Cython.Utils
-from Cython.TestUtils import CythonTest, write_file, fresh_cythonize
+from Cython.TestUtils import (CythonTest, write_file, fresh_cythonize,
+                              relative_lines_from_file)
```

Comment:
If we can afford the extra newline in other parts of this PR, we can afford it here, too.
```suggestion
from Cython.TestUtils import (
    CythonTest, write_file, fresh_cythonize, relative_lines_from_file)
```

Reply:
also [sorted](https://github.com/cython/cython/pull/4241/commits/b771523e583377b68ad21f31f4fd7ed6b683d169)